modules/nodes-nodes-modifying-resources.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
```

Comment:
obsolete, replaced by the in-development kubelet-config-controller that will be part of the MCO.  just hold on documenting kubelet configuration for now.